### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by iLtc**
This PR updates the Readme file so (hopefully) it is easier to follow.

You can preview the updated doc here: https://github.com/iLtc/shopcarts/blob/update-readme/README.md

## Checklist:

- [x] I have updated documentation to reflect this change
- [x] I have attached this PR to an issue/user story

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,216 +87,439 @@ honcho start
...
+```
+
+#### Successful Response
 
 ```json
 {
-    "amount": 4,
+    "amount": 3,
+    "create_time": "2020-11-06T20:48:07.731641",
+    "id": 4,
```
Comment (KHafner) [inline README.md:417]: id should be 3

**Review by KHafner — APPROVED** [review]
Great, thanks

**Review by nonococoleo — APPROVED** [review]
THX！

File: README.md (hunk 1 above)
Comment (iLtc) [reply README.md]: Thanks for catching that!

**Comment by KHafner** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/b5c2987e-b3d1-4601-9f28-34419832137d?env_id=ibm:yp:us-south "nyu-shopcart-service-f20-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/c869c127-ceda-423d-80b8-f74ee52b7378 "nyu-shopcart-service-f20-pipeline") deployed [shopcarts](https://github.com/NYU-DevOps2020-shopcarts/shopcarts/releases/tag/deploy-ibm-yp-us-south-kfh226-nyu.edu-dev-20201117-175836 "deploy-ibm-yp-us-south-kfh226-nyu.edu-dev-20201117-175836") to [dev](https://console.bluemix.net/apps/4aa42608-55fd-40d6-b27b-d99aa18a0658?env_id=ibm:yp:us-south "ibm:yp:us-south:kfh226@nyu.edu:dev"), including commit ded0ae6440f947dee65f4b57b668a2d7157612d0

**Comment by KHafner** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/b5c2987e-b3d1-4601-9f28-34419832137d?env_id=ibm:yp:us-south "nyu-shopcart-service-f20-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/c869c127-ceda-423d-80b8-f74ee52b7378 "nyu-shopcart-service-f20-pipeline") deployed [shopcarts](https://github.com/NYU-DevOps2020-shopcarts/shopcarts/releases/tag/deploy-ibm-yp-us-south-kfh226-nyu.edu-prod-20201208-144028 "deploy-ibm-yp-us-south-kfh226-nyu.edu-prod-20201208-144028") to [prod](https://console.bluemix.net/apps/ac06a6b3-040d-43d7-b290-6bfbba3f8c41?env_id=ibm:yp:us-south "ibm:yp:us-south:kfh226@nyu.edu:prod"), including commit ded0ae6440f947dee65f4b57b668a2d7157612d0

**Pull request merged**